fission/function.go

```diff
@@ -118,7 +118,7 @@ func fnCreate(c *cli.Context) error {
 	client := getClient(c.GlobalString("server"))
 
 	if len(c.String("package")) > 0 {
-		fatal("--package is deprecated, please use --deploy instead.")
+		fatal("--package is deprecated and will be remove in the next release, please use --deploy instead.")
```

Comment:
It's been a fatal error and deprecated for a few releases already, right?  We can just remove this one.